src/components/Card/Card.vue

```diff
@@ -1,28 +1,43 @@
 <template>
   <article class="location-card" :id="id">
-    <div class="button-container">
-      <button class="button button-refresh" type="button" name="button" v-on:click.prevent="refreshLocation">🔄</button>
-      <button class="button" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    <div v-if="currentLocation">
+      <p>Current Location</p>
     </div>
-    <h2>{{ locale }}</h2>
+    <div v-else class="button-delete-container">
+      <button class="card-button button-delete" type="button" name="button" v-on:click.prevent="deleteCard">X</button>
+    </div>
+    <h2>{{ formatLocation() }}</h2>
     <img :src="require(`../../assets/${formatWeatherIcon()}.png`)" :alt="weatherIconAlt" class="weather-icon">
     <h3>Weather</h3>
```

Comment:
I added the :alt="weatherIconAlt" into the image tag while resolving merge conflicts, this should be working without issue!